CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # TBD
 * Drop default loglevel for initializer & controller down to DEBUG
 * Upgrade controller Docker image to allow for a Docker network per test
+* Run tests in parallel!
+* Removed references to `fiveNode` in testsuite (because these networks are no longer five-node, and the network size is less important than whether it's staking or not)
```

Comment:
can we add that we're removing the "GetValidators" test to the changelog? it's significant enough change because ava labs CI is currently calling that test by name

Reply:
Yep: https://github.com/ava-labs/gecko/pull/272/files